resident/resident-service/src/main/java/io/mosip/resident/dto/AuthorizedScopesDto.java

```diff
@@ -40,5 +40,7 @@ public class AuthorizedScopesDto {
 	private List<String> getServiceAuthHistoryRoles;
 
 	private List<String> getServiceRequestUpdate;
+
+	private List<String> getCredentialRequestStatus;
```

Comment:
Correct the property like `getCheckAidStatus`

Reply:
Done.